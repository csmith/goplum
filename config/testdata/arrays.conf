defaults {
    ints = [1, 2, 3, 4,]
    strings = ["a", "b", "c", ",", "d"]
    floats = [1.3, 1.4, 1.5]
    durations = [1s, 3m, 1d]
    bools = [
        true, # Comment here
        false,yes,
        no
    ]
}
