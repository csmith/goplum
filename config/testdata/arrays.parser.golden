{
  "DefaultSettings": {
    "bools": [
      true,
      false,
      true,
      false
    ],
    "durations": [
      1000000000,
      180000000000,
      86400000000000
    ],
    "floats": [
      1.3,
      1.4,
      1.5
    ],
    "ints": [
      1,
      2,
      3,
      4
    ],
    "strings": [
      "a",
      "b",
      "c",
      ",",
      "d"
    ]
  },
  "AlertBlocks": null,
  "CheckBlocks": null,
  "PluginSettings": null
}